### PR TITLE
E-puck standardization

### PR DESCRIPTION
**Pull request by remidhum**
Standardization of the E-puck page based on the Pioneer 3-AT and 3-DX.

Preview: https://www.cyberbotics.com/doc/guide/using-the-e-puck-robot?version=enhancement-epuck-standardization

### CODE REVIEW

**Comment by remidhum** [comment]
👍

File: guide/using-the-e-puck-robot.md [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+%figure "The e-puck robot"
+
+![e-puck.png](images/e-puck.png)
+
+%end
+
 E-puck is a miniature mobile robot originally developed at EPFL for teaching purposes by the designers of the successful Khepera robot.
 The hardware and software of e-puck is fully open source, providing low level access to every electronic device and offering unlimited extension possibilities.
+
+The model includes support for the differential wheel motors (encoders are also simulated, as position sensors), the infra-red sensors for proximity and light measurements, the accelerometer, the camera, the 8 surrounding LEDs, the body and front LEDs; the other e-puck devices are not yet simulated in the current model.
```
Comment (fabienrohrer) [inline guide/using-the-e-puck-robot.md]: Why not put this paragraph into the `E-puck model` section?

Comment (fabienrohrer) [reply guide/using-the-e-puck-robot.md]: You could also add the "ground sensors extension" (this is optional), and the bluetooth communication device (This is modeled using Emitter / Receiver devices)

File: guide/using-the-e-puck-robot.md [hunk 2]
```diff
@@ -40,29 +48,36 @@ E-puck is equipped with a large number of devices, as summarized in [this table]
...
-Besides standard e-puck devices, the model also provides two slots that can be used to extend the functionality and to equip the robot with additional devices.
-In particular, the ground sensors module extension of the real e-puck robot is modeled in Webots to provide 3 optional infra-red sensors pointing to the ground in front of the robot.
-The [table](#devices-names) displays the names of the simulated devices which are to be used as an argument of the `wb_robot_get_device` function (see the Robot section of the [Reference Manual](http://www.cyberbotics.com/reference/)).
+The standard model of the e-puck is provided in the "E-puck.proto" PROTO file which is located in the "WEBOTS\_HOME/projects/robots/e-puck/protos" directory of the Webots distribution (see also "E-puckDistanceSensor.proto" PROTO file and "E-puckGroundSensors.proto" PROTO file); you will find complete specifications in it.
```
Comment (fabienrohrer) [inline guide/using-the-e-puck-robot.md]: These paths are wrong:
`WEBOTS_HOME/projects/robots/e-puck/` => `WEBOTS_HOME/projects/robots/gctronic/e-puck/`

File: guide/using-the-e-puck-robot.md [hunk 3]
```diff
@@ -40,29 +48,36 @@ E-puck is equipped with a large number of devices, as summarized in [this table]
...
+The two PROTO fields `groundSensorSlot` and `turretSlot` have been included in the simulation model in order to support extension modules.
+In particular, the ground sensors module extension of the real e-puck robot is modelled in Webots to provide 3 optional infra-red sensors pointing to the ground in front of the robot.
```
Comment (fabienrohrer) [inline guide/using-the-e-puck-robot.md]: You could add here that the gound sensor extension is modeled using the `E-puckGroundSensors` PROTO, therefore, adding this PROTO into the  `groundSensorSlot` slot is doing the job. (but try to formulate this better :-) )

File: guide/using-the-e-puck-robot.md [hunk 4]
```diff
@@ -111,12 +113,12 @@ The `wb_motor_set_velocity` and `wb_position_sensor_get_value` functions allow y
 %end
 
 The forward direction of the e-puck is given by the negative *z*-axis of the world coordinates.
-This is also the direction the eye of the camera is looking to; in keeping with the VRML97 standard, the direction vector of the camera is pointing in the opposite direction, namely the direction of the positive *z*-axis.
+This is also the direction in which the camera eye is looking; to comply to the VRML97 standard, the direction vector of the camera is pointing in the opposite direction, namely the direction of the positive *z*-axis.
```
Comment (fabienrohrer) [inline guide/using-the-e-puck-robot.md]: Minor: I'm wondering if explaining the reason of the axis choice is important enough to document it.

File: guide/using-the-e-puck-robot.md [hunk 5]
```diff
@@ -300,3 +275,51 @@ This error can be fixed simply by installing the following packets from a termin
 ```sh
 sudo apt-get install gcc-4.6-base:i386 libgcc1:i386 libc6:i386
 ```
+
+### Samples
+
+Several simulation examples are located in the "WEBOTS\_HOME/projects/robots/e-puck/worlds" directory of the Webots distribution.
+
+The directory "WEBOTS\_HOME/projects/samples/curriculum" contains a rich collection of simulations involving the e-puck robot.
```
Comment (fabienrohrer) [inline guide/using-the-e-puck-robot.md]: I'm wondering it the Curriculum would not deserve a section (h2) rather. It seems this important URL is missing: `https://en.wikibooks.org/wiki/Cyberbotics'_Robot_Curriculum`

File: guide/using-the-e-puck-robot.md (hunk 4 above)
Comment (remidhum) [reply guide/using-the-e-puck-robot.md]: Maybe @omichel has an idea

File: guide/using-the-e-puck-robot.md (hunk 1 above)
Comment (remidhum) [reply guide/using-the-e-puck-robot.md]: A short description of the model is present in the last 3 standardized documentation pages.
https://www.cyberbotics.com/doc/guide/using-the-pioneer-3-at-robot

File: guide/using-the-e-puck-robot.md (hunk 4 above)
Comment (omichel) [reply guide/using-the-e-puck-robot.md]: I would simply remove "to comply with the VRML97 standard, ".

**Comment by remidhum** [comment]
👍

**Comment by omichel** [comment]
The devices table does not display device names in a consistent way, sometime we use single quotes, sometimes not.

**Comment by remidhum** [comment]
👍

**Comment by omichel** [comment]
The e-puck_sensors_and_leds.png image is misleading because the sensor rays (or what looks like sensor rays) are not the same length whereas in reality they have the same range.

**Comment by remidhum** [comment]
👍

**Comment by fabienrohrer** [comment]
I allow myself to merge this PR as Remi is no more present.

**Pull request merged**